### PR TITLE
complete readme template

### PR DESCRIPTION
**Pull request by getchoo**
this is a draft for now since of course i will need feedback, but i think this is a good step towards getting this into the org

so far i've added instructions, changed most links to point to catppuccin/nix, and added a bit to the q&a

### CODE REVIEW

**Comment by Stonks3141** [comment]
I’m going to undraftify this since it is complete and just needs review.

File: README.md [hunk 1]
```diff
@@ -36,20 +36,109 @@
...
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Import the NixOS and [home-manager](https://github.com/nix-community/home-manager) modules
+
+<details>
+<summary>With Flakes</summary>
```
Comment (Stonks3141) [inline README.md:42]: Should we have separate instructions for HM only?

Comment (getchoo) [reply README.md:42]: you mean for a standalone install?

Comment (Stonks3141) [reply README.md:42]: I mean if someone is using Nix and HM on Ubuntu or something and doesn't want the NixOS module.

Comment (getchoo) [reply README.md:42]: yeah that'd be a [standalone install](https://nix-community.github.io/home-manager/index.html#sec-install-standalone)

File: README.md [hunk 2]
```diff
@@ -36,20 +36,135 @@
...
+  programs.starship = {
+    enable = true;
+    catppuccin.enable = true;
+  };
+}
+```
+
+</details>
 
 <!-- this section is optional -->
```
Comment (Stonks3141) [inline README.md:158]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -36,20 +36,135 @@
...
+        catppuccin.homeManagerModules.catppuccin
+      ];
+    };
+  };
+}
+```
+
+</details>
+
+<details>
```
Comment (Stonks3141) [inline README.md:88]: This `<details>` tag isn't closed, so the entire rest of the README is hidden in it.

File: README.md [hunk 4]
```diff
@@ -36,20 +36,135 @@
...
+  imports = [
+    <catppuccin/modules/home-manager>
+  ];
+
+  home.username = "user";
+  home.homeDirectory = "user";
+  programs.home-manager.enable = true;
+}
+```
+
```
Comment (Stonks3141) [inline README.md:128]:
```suggestion

</details>

```

File: README.md [hunk 5]
```diff
@@ -36,20 +36,135 @@
 
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Import the NixOS and [home-manager](https://github.com/nix-community/home-manager) modules
```
Comment (Stonks3141) [inline README.md:39]:
```suggestion
1. Import the [NixOS](https://nixos.org) and [home-manager](https://github.com/nix-community/home-manager) modules
```
I think it looks more consistent with both as a link, even though https://nixos.org is linked elsewhere.

**Review by Stonks3141 — CHANGES_REQUESTED** [review]

**Review by Stonks3141 — APPROVED** [review]

**Pull request merged**